### ISSUE TITLE
Teensy 3.5 ports

### ISSUE THREAD

**Issue by dimitre** [FastLED/FastLED#364]
It would be great to support Teensy 3.5 ports >= 24.
I'm testing with Adafruit_DotStar and they are already working there.
Thank you!

**Comment by Mad-Onion** [comment]
@dimitre how did you get Teensy 3.5 with FastLED working? When using FastLED 3.1.5, Teensyduino 1.3.1 and Arduino 1.6.12 I get different errors in the blink-example:
- No pin/port mappings found
- no matching function for call to 'CFastLED::addLeds(CRGB [1], int)'

**Comment by dimitre** [comment]
@Mad-Onion it worked using branch 339, as you can follow on this thread:
https://github.com/FastLED/FastLED/issues/339

**Comment by Mad-Onion** [comment]
@dimitre thank you!

Is there a roadmap for a T3.5 port?

**Comment by focalintent** [comment]
No, everything is on an indefinite hold right now while @kriegsman and I deal with a death in the family.

**Comment by kcouck** [comment]
Isn't this the same issue as #361?

**Comment by focalintent** [comment]
Someone submitted 3.5/3.6 support.
